developer-docs/README.md

```diff
@@ -98,6 +102,10 @@ Do this for all supported versions of Python. You can check your virtual env
 Python version using `sematic version` (as well as the Server and Client
 version).
 
+At this point, you should also deploy to one of the dev environments and execute
```

Comment:
External users won't have access to out dev envs, so we should keep the wording generic.
Also, I'd like to explicitly specify deploying using the Helm charts.
```suggestion
At this point, you should also deploy to a cloud Dev environment using the Helm chart and execute
```

Reply:
I don't think external users should ever do a release 😆 . But sure, happy to make this change.